com.github.petebuffon.yafc.yml

```diff
@@ -0,0 +1,24 @@
+app-id: com.github.petebuffon.yafc
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: YAFC
+finish-args:
+  - "--socket=wayland"
+  - "--device=dri"
+  - "--socket=x11"
+  - "--share=ipc"
+  - "--filesystem=~/.factorio/mods:ro"
+  - "--filesystem=~/.steam/steam/steamapps/common/Factorio:ro"
+modules:
+  - name: yafc
+    buildsystem: simple
+    build-commands:
+      - install -Dp -m 755 YAFC /app/bin/YAFC
+      - install -Dp -m 644 com.github.petebuffon.yafc.desktop /app/share/applications/com.github.petebuffon.yafc.desktop
+    sources:
+      - type: "file"
+        url: "https://github.com/petebuffon/yafc/releases/download/v0.5.8/YAFC"
```

Comment:
so is this x86_64?

You need to https://github.com/flathub/flathub/wiki/App-Maintenance#only-build-on-x86_64

Reply:
Yex x86_64, flathub.json file added